airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -149,6 +155,44 @@ def get_task_failure_counts():
             TaskFail.task_id,
         )
 
+def get_xcom_params(task_id):
+    #get latest task_id, join to xcom table
```

Comment:
Could we add docstring for the function?

Reply:
yes